bindings/jupyter/docs/manifold.md

```diff
@@ -53,15 +54,15 @@ data = {
   x:     [...],         # feature data
   yPred: [[...], ...]   # prediction data
   yTrue: [...],         # ground truth data
-}
+};
 ```
 
 Each element in these lists represents one data point in your evaluation dataset, and the order of data instances in `x`, `yPred` and `yTrue` should all match.
 Recommended instance count for each of these datasets is 10000 - 15000. If you have a larger dataset that you want to analyze, a random subset of your data generally suffices to reveal the important patterns in it.
 
-##### `x` (list | ndarray, required):
+##### `x` (list | ndarray | data_frame, required):
+A list/ndarray/data_frame of instances with features. Example (2 data instances):
```

Comment:
Nit: Use the library + type name, like `pandas.DataFrame` and `numpy.ndarray`

Reply:
addressed.